org.civicrm.reqangular/src/common/services/session.js

```diff
@@ -42,4 +42,4 @@
       };
     }]);
   });
-})(CRM, Drupal);
+})(CRM, window.Drupal);
```

Comment:
After this change I think we need change in all places where `session.contact_id` is used. right?